components/printful_oauth/actions/create-order/create-order.mjs

```diff
@@ -0,0 +1,138 @@
+import printful_oauth from "../../printful_oauth.app.mjs";
+
+export default {
+  name: "Create Order",
+  version: "0.0.1",
+  key: "printful_oauth-create-order",
+  description: "Creates an order. [See docs here](https://developers.printful.com/docs/#operation/createOrder)",
+  type: "action",
+  props: {
+    printful_oauth,
+    externalId: {
```

Comment:
Since you're using the same props on both components, why not reuse it by importing from a common file?